src/aks-preview/HISTORY.rst

```diff
@@ -12,8 +12,11 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 
+0.5.120
++++++++
 * GA file, blob csi driver and snapshot controller.
 * GA Azure Dedicated Host.
+* GA KMS.
```

Comment:
I am afraid that "GA KMS" is confusing. If the customer checks https://github.com/MicrosoftDocs/azure-docs-cli/blob/main/docs-ref-conceptual/release-notes-azure-cli.md#august-02-2022 and this file, they will get two different dates for KMS GA.

Reply:
Sure, updated to clarify we will reuse the code from azure-cli/acs.